.github/workflows/dashboard-v1-testnet.yml

```diff
@@ -88,6 +89,7 @@ jobs:
             tbtc-contracts-version = github.com/keep-network/tbtc/solidity#version
             tbtc-v2-contracts-version = github.com/keep-network/tbtc-v2/solidity#version
             coverage-pools-version = github.com/keep-network/coverage-pools#version
+            threshold-contracts-version = github.com/threshold-network/solidity-contracts#version
```

Comment:
I'm 90% sure that we need to do some modifications in https://github.com/keep-network/ci/blob/main/config/config.json and https://github.com/keep-network/ci/tree/main/actions/upstream-builds-query to make this work. I will verify that and create the needed PRs in the `ci` repo.

Reply:
Also, we should modify the actions in the `ci` repo to make the `@threshold-network/solidity-contracts` automatically deploy somewhere after `keep-core` and before `coverage-pools` (see https://coda.io/d/Building-Keep_d-fmEgBNFVH/Current-CI-process_su1ww#_luqjk). I will work on that as well.